### ISSUE TITLE
code snippets and examples in manual need to use NIAC2014 attributes

### ISSUE THREAD

**Issue by prjemian** [nexusformat/definitions#443]
* add NIAC2014 attributes to NeXus history
* revise code snippets and data examples to use default/default/signal/axes/*_indices attributes
* consider adding a flowchart graphic to identify default data for plotting (useful for code developers who might encounter any of the three methods)

**Comment by prjemian** [comment]
in ``napi.rst`` there are references to NAPI F90 Utility Routines

* ``NXUfindsignal`` : Searches the currently open group for a data item with the specified ``SIGNAL`` attribute.       
* ``NXUfindaxis` : Searches the currently open group for a data item

Will the NAPI F90 Utility Routines be refactored to consider the NIAC2014 attributes?  Handle separately in the **code** repository.

**Comment by prjemian** [comment]
in **NeXus/definitions/manual/source/examples**

these instances of *signal* need to be resolved by someone familiar with the code:
```
napi-example.c:            NXputattr (file_id, "signal", &i, 1, NX_INT32);
napi-example.f77:              status = NXputattr (file_id, 'signal', 1, 1, NX_INT32)
napi-example.f90:            if (NXputattr (file_id, "signal", 1) /= NX_OK) stop
nxh5write.c: *            @signal=1
nxh5write.c:  int counts[LENGTH], i, rank, signal;
nxh5write.c:   * set the signal=1 attribute 
nxh5write.c:  attid = H5Acreate(dataid,"signal", atttype, atts, H5P_DEFAULT);
nxh5write.c:  signal = 1;
nxh5write.c:  H5Awrite(attid, atttype, &signal);
```

**Comment by prjemian** [comment]
There is a note that the f77 interface will no longer be developed.

**Comment by prjemian** [comment]
need to regenerate the simpl3D data with the revised code

**Comment by prjemian** [comment]
in **rules.rst** the *Simple scan with area detector* example should be checked for the ``@axes`` attribute. the 2nd and 3rd dimensions have not available axes datasets to be plotted and should be assumed that any client will understand this.

**Comment by prjemian** [comment]
Also, scour for missing ``_indices`` attributes in various examples and snippets

**Comment by prjemian** [comment]
at this point, all base_classes and application definitions have been refactored to use new default/default/signal/axes/AXISNAME_indices attributes
contributed definitions need to be refactored

**Comment by prjemian** [comment]
all contributed definitions with NXentry and/or NXdata groups have been refactored

**Comment by prjemian** [comment]
need to add a few graphic files to illustrate the different ways to find the plottable data, check up on #436, then ready for pull request

* #443 : use NIAC2014 attributes in the manual throughout
* #442 : dropped my_lib from h5py examples
* #444 : reversed presentation of chronology in history file
* #445 : removed entry attribute from NXentry
* #436 : still TODO
* #437 : TODO pending revision of nexpy/nexusformat code

**Comment by zjttoefs** [comment]
I'm in agreement with all the changes made until yesterday to examples and code snippets. I had a glance yesterday and that looks very good.
The changes to the application definitions are going a bit far to do them without discussion. Especially since we're still none the wiser how to handle versioning there.

I would propose I merge the non application definition changes and put the other ones on a separate branch to look at individually.

**Comment by prjemian** [comment]
I'll issue a PR when this branch is ready

**Comment by prjemian** [comment]
see PR #448

**Comment by zjttoefs** [comment]
Discussed in Telco. Should be merged to master.
Thanks Pete!

**Comment by prjemian** [comment]
fixed in #453
